.github/workflows/generate_package_wiki.yml

```diff
@@ -0,0 +1,102 @@
+name: Generate Package Wiki
+
+on:
+  schedule:
+    # Runs daily at a weird time (02:50 UTC) to avoid delays during periods of
+    # high loads of GitHub Actions workflow runs.
+    - cron: '50 2 * * *'  
+  workflow_dispatch:  # Allow manual triggering
+
+jobs:
+  generate-wiki:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Checkout Wiki Repo
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.repository }}.wiki
+          path: wiki
+          token: ${{ secrets.REPO_TOKEN }}
+          fetch-depth: 0
+
+      - name: Generate Wiki Content
+        id: generate-content
+        shell: pwsh
+        run: |
```

Comment:
I think we should extract this script to a file so that we can run it locally and to simplify the code in the action.